d3a_api_client/rest_device.py

```diff
@@ -80,6 +80,20 @@ def unselect_aggregator(self, aggregator_uuid):
                                           "device_uuid": self.device_id}, self.jwt_token)
         self.active_aggregator = None
 
+    @logging_decorator('pv_power_forecast')
+    def set_pv_power_forecast(self, pv_power_forecast):
+        transaction_id, posted = self._post_request('set_pv_power_forecast',
+                                                    {"pv_power_forecast": pv_power_forecast})
+        if posted:
+            return self.dispatcher.wait_for_command_response('set_pv_power_forecast', transaction_id)
+
+    @logging_decorator('set_load_power_forecast')
+    def set_load_power_forecast(self, pv_power_forecast):
```

Comment:
Minor, but argument name should be `load_power_forecast`. Also a unit suffix would help in both functions' arguments.